### PR TITLE
Adapt zksync era testing

### PR DESCRIPTION
**Pull request by juan518munoz**
This PR is aimed to give compatibility with the zksync-era testsuite, it needs to be merged alongside [this other PR](https://github.com/lambdaclass/zksync-era/pull/229).

### CODE REVIEW

**Comment by MarcosNicolau** [comment]
We might have to account for pr #200 for the adaptation first, since it changes the ContractStorage and StateStroge for a single Storage trait.

**Comment by juan518munoz** [comment]
Yes, let's wait for that PR before we merge this one.

**Review by fkrause98 — APPROVED** [review]

**Review by gianbelinche — APPROVED** [review]

**Pull request merged**
